### DISCUSSION TITLE
Customize jedi-language-server features

### DISCUSSION THREAD

**Discussion by jgarte** [joaotavora/eglot#1059] [Q&A]
Hi,

How is possible to customize jedi-language-server features similar to these but with eglot?

The settings in the link are not [workspace configurations](https://github.com/joaotavora/eglot#eglot-workspace-configuration) correct?

https://github.com/fredcamps/lsp-jedi/blob/master/lsp-jedi.el#L177

**Comment by doolio** [comment]
What you linked to seems to be a package to integrate the [jedi-language-server](https://github.com/pappasam/jedi-language-server) with `lsp-mode` which is the other LSP client for Emacs. `eglot` has been tested with the `jedi-language-server` as per its [README](https://github.com/joaotavora/eglot#connecting-to-a-server). It seems that language server supports [`initialization`](https://github.com/pappasam/jedi-language-server#configuration) but not sure if it supports `workspace/configuration`. I believe `eglot` supports both methods.

**Reply by jgarte** [reply]
> What you linked to seems to be a package to integrate the [jedi-language-server](https://github.com/pappasam/jedi-language-server) with lsp-mode which is the other LSP client for Emacs.

I know. I linked that to make a comparison of what's available in lsp-mode versus eglot

> has been tested with the jedi-language-server as per its [README](https://github.com/joaotavora/eglot#connecting-to-a-server)

@doolio I was the person who wrote that and I haven't tested all the jedi-language-server features 💯

**Comment by doolio** [comment]
Ah I see. Issue #940 may hint at how you can set such server options.
